src/Components/Projects/Projects.tsx

```diff
@@ -1,7 +1,144 @@
+import noData from '../../assets/noData.png'
+ import avatar from '../../assets/avatar.png'
+ import Datano from '../../assets/DataNo.svg'
+import { Link, useNavigate } from 'react-router-dom';
+import { Modal } from 'react-bootstrap';
 export default function Projects() {
+const {baseUrl,reqHeaders}:any=useContext(AuthContext)
+let [allProjs,setAllProjs]=useState([])
+let [itemId,setItemId]=useState(0)
+const navigate= useNavigate()
+const[modalState,setModalState]=useState("close")
+const handleClose = () => setModalState("close");
+
+  const showAllProjects =()=>{
+    axios.get(`${baseUrl}/Project`,{headers:reqHeaders}).then((response)=>{
+      console.log(response);
+      setAllProjs(response?.data)
+      
+    }).catch((error)=>{
+      console.log(error);
+      
+    })
+
+  }
+  const navToAddPro =()=>{
+    navigate('/dashboard/Add-pro')
+  }
+  const editShow =(id)=>{
+    
+    navigate(`/dashboard/edit-pro/${id}`)
+    setItemId(itemId)
+  }
+  const showDeleteModel = ()=>{
+   
+    
+    setModalState("delete-modal")
+  }
+  const viewPro = (id)=>{
+    navigate(`/dashboard/view-pro/${id}`)
+  }
+  useEffect(() => {
+    showAllProjects()
+    
+  }, [])
+  
   return (
-    <div>Projects</div>
+    <>
+     <Modal show={modalState==='delete-modal'} onHide={handleClose}>
+       
+       <Modal.Body>
+       <div className="delete-container">
+          <div className="icons text-end">
+        <i onClick={handleClose} className="fa-regular fa-circle-xmark text-danger "></i>
+          </div>
+          <div className="text-center">
+            <div className="text-center">
+            <img className=' '  src={Datano} alt="msg-NoData" />
+            </div>
+            <h5 className='py-3'> Are you sure to Delete this item ? </h5>
+          </div>
+         
+
+         
+          <div className="delete-btn text-end">
+            <button  className='text-white bg-danger btn btn-outline-danger   border-danger rounded-2  '>Delete This Item </button>
+          </div>
+        </div>
+   
+   
+       </Modal.Body>
+      
+     </Modal>
+ <div className="container bg-white rounded-3 px-3">
+  <div className="col-md-12 trans-head">
+    <div className="top d-flex justify-content-between ">
+      <h4>Projects</h4>
+      <button onClick={navToAddPro} className='btn btn-warning bg-warning text-white'> <i className='fa-plus'></i> Add New Project </button>
+    </div>
+
+  </div>
+  <div className="col-md-12 ">
+    <input type="text" placeholder='SearchFleets 'className='rounded-4 border-1 mb-4 pro' />
+ 
+
+
+    <Table striped bordered hover>
+      <thead >
+        <tr className='bg text-center' >
+          <th>Title</th>
+          <th>Manager</th>
+          <th>Description</th>
+          <th>Image</th>
+          <th>Date Created</th>
+          <th>
+        
+          </th>
+
+
+        </tr>
+      </thead>
+      <tbody className='text-center'>
+        {allProjs.length>0?allProjs.map((pro)=>(<tr>
```

Comment:
Well here you forgot to add the "key" attribute to the "tr" element, so that react and keep an eye on every change that may happen to each different in the list you want to display